Speak/default.jx3dat

```diff
@@ -14,6 +14,70 @@ local FORCE_TYPE = {
   CHANG_GE  = 22, -- 长歌
   BA_DAO    = 23, -- 霸刀
 }
+
+
 data = {
-  [FORCE_TYPE.WAN_HUA] = { 101--[[握针]], 137--[[局针]], 138--[[提针]], 142--[[长针]], 141--[[毫针]], 14664--[[彼针]], 139--[[锋针]], 2654--[[利针]], 143--[[大针]] },
+  [FORCE_TYPE.SHAO_LIN] = {
+  	[200] = { "hit" }, --[[般若诀]]
+  	[258] = { "hit" }, --[[舍身决]]	
+  },
+  [FORCE_TYPE.WAN_HUA] = { 
+  	[139] = { "hit", "casting" }, --[[锋针]]
```

Comment:
When comments appears at the end of a line just use `-- comments` instead of `--[[comments]]`